packages/api/src/webviewWindow.ts

```diff
@@ -122,17 +126,14 @@ class WebviewWindow {
   /**
    * Get an instance of `Webview` for the current webview.
    */
+
   static getCurrent(): WebviewWindow {
     return getCurrentWebviewWindow()
   }
 
-  /**
-   * Gets a list of instances of `Webview` for all available webviews.
-   */
   static async getAll(): Promise<WebviewWindow[]> {
     return getAllWebviewWindows()
   }
-
```

Comment:
why remove this newline? run `pnpm format` in repo root to fix these formatting issue